c2corg_ui/static/js/alertservice.js

```diff
@@ -120,11 +120,16 @@ app.Alerts.prototype.formatErrorMsg_ = function(response) {
   if (len > 1) {
     var msg = '<ul>';
     for (var i = 0; i < len; i++) {
-      msg += '<li>' + this.filterStr_(errors[i]['description']) + '</li>';
+      if (errors[i]['description'] && errors[i]['name']) {
+        msg += '<li><span translate>' + this.filterStr_(errors[i]['description']) +
+                        '</span><span translate> ' + this.filterStr_(errors[i]['name']) +
```

Comment:
I don't think the "translate" directive are needed here since ``this.filterStr_`` already translates the strings:
https://github.com/c2corg/v6_ui/blob/7c50d4b7b4fff12a8eb4b4d2b49efd9d18a87e70/c2corg_ui/static/js/alertservice.js#L141-L144